packages/ibm-products/src/components/Datagrid/useDisableSelectRows.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed Materials - Property of IBM
+ * 5724-Q36
+ * (c) Copyright IBM Corp. 2020
+ * US Government Users Restricted Rights - Use, duplication or disclosure
+ * restricted by GSA ADP Schedule Contract with IBM Corp.
+ */
+
+import { Hooks, Row, RowPropGetter, TableRowProps } from 'react-table';
+import { DataGridTableInstance, DatagridRow } from './types';
+
+interface PropGetterMeta {
+  instance?: DataGridTableInstance;
+  row?: Partial<Row<any> & DatagridRow<any>>;
+}
+
+const useDisableSelectRows = (hooks: Hooks) => {
+  updateSelectAll(hooks);
+  updatePageSelectAll(hooks);
+
+  const getRowProps: RowPropGetter<any> = (
+    props: Partial<TableRowProps>,
```

Comment:
We are creating this Partial type multiple times in this file. I think, it would be better to declare a `PartialTableRowProps` type in `packages/ibm-products/src/components/Datagrid/types/index.ts` file and reuse it where ever required;
`type PartialTableRowProps = Partial<TableRowProps>;`

Reply:
I think that is not necessary, because `Partial<...>` usage is normal, also this is more readable, right?